src/assets/toolkit/scripts/lib/component/slideshow.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+import {u} from 'umbrellajs';
+export class Slideshow {
+  constructor (element, {
+    slideHolder = element.querySelector('.js-Slideshow-slides'),
+    slides = element.querySelectorAll('.js-Slideshow-slide'),
+    nextTrigger = element.querySelector('.js-Slideshow-next'),
+    prevTrigger = element.querySelector('.js-Slideshow-prev'),
+    currentCountElement = element.querySelector('.js-Slideshow-current'),
+    totalCountElement = element.querySelector('.js-Slideshow-total'),
```

Comment:
> Also, it does appear as though there are some excessive calls to element.querySelectorAll() in each case where we subsequently wrap the elements with Umbrella's u() function.

Ah, yes, great point @erikjung.

Related: I was wondering...when using jQuery, a pattern arised to use the `$` as a variable name prefix to denote if the element was a jQuery wrapped object.

Might we want to follow a similar pattern here? Perhaps adding `u` as a prefix?

For example:
```js
const uSlideHolder = u(slideHolderSelector);
```